js/app/src/Blocks.svelte

```diff
@@ -448,15 +449,25 @@
 					on:click={() => {
 						set_api_docs_visible(!api_docs_visible);
 					}}
-					class="show-api"
+					class="footer-link"
 				>
 					Use via API <img src={api_logo} alt="" />
 				</button>
 				<div>·</div>
 			{/if}
+			{#if true}
```

Comment:
I think there should be a way to not display this by the app author. Some spaces are not meant to be duplicated (they use a custom backend like stable diffusion or whisper-jax which doesn't transfer on udplication) or app author just doesn't want to include it.

Reply:
Added parameter to remove.